pkg/option/flags.go

```diff
@@ -89,7 +89,7 @@ const (
 	KeyEnablePodInfo          = "enable-pod-info"
 	KeyEnableTracingPolicyCRD = "enable-tracing-policy-crd"
 
-	KeyExposeKernelAddresses = "expose-kernel-addresses"
+	KeyExposeAddresses = "expose-addresses"
```

Comment:
As we split everything in kernel-stack and user-stack, might be useful to also split this here. Or use a name like `expose-stack-addresses`. The concern was mostly about kernel address in the first place so that would be better if the idea of exposing _kernel_ address is still around.